### PR TITLE
PMM-7874 UI tests migration

### PR DESCRIPTION
**Pull request by yurkovychv**
https://jira.percona.com/browse/PMM-7874

### CODE REVIEW

**Review by puneet0191 — APPROVED** [review]
Looks good, thank you..you could also update the readme file with instructions on how to execute tests locally, maybe do it in seperate PR.

**Comment by yurkovychv** [comment]
> Looks good, thank you..you could also update the readme file with instructions on how to execute tests locally, maybe do it in seperate PR.

sure will do

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+.idea
```
Comment (atymchuk) [inline .gitignore:1]: Pls add `.vscode` as well

**Review by atymchuk — APPROVED** [review]

**Comment by atymchuk** [comment]
Great job!

File: .gitignore (hunk 1 above)
Comment (yurkovychv) [reply .gitignore:1]: done, thanks

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**